src/common/helpers/requestMetaData/approvedIdentifiersTableParser.ts

```diff
@@ -0,0 +1,91 @@
+import fs from "fs";
+import path from "path";
+import https from "https";
+
+main();
+
+async function main() {
+  // first download the table of approved identifiers from our docs
+  const url =
+    "https://raw.githubusercontent.com/UMAprotocol/docs/master/docs/uma-tokenholders/approved-price-identifiers.md";
+  const markdownPath = path.join(__dirname, "./approvedIdentifiersTable.md");
+  const jsonPath = path.join(__dirname, "./approvedIdentifiersTable.json");
+
+  await downloadFile(url, markdownPath);
+
+  // read and parse the markdown file from GitHub
+  const contents = fs.readFileSync(markdownPath, "utf-8");
+  const lines = contents
+    .split("\n")
+    // ignore the first 8 lines (headmatter and table headers)
+    .slice(8);
+  const parsedLines = JSON.stringify(lines.map(parseLine));
+
+  // write the parsed lines to a json file
+  fs.writeFile(jsonPath, parsedLines, (err) => console.error(err));
+
+  // remove the markdown file
+  fs.unlinkSync(markdownPath);
+}
+
+function parseLine(line: string) {
+  const dividerIndices: number[] = [];
+
+  for (let i = 0; i < line.length; i++) {
+    if (line[i] === "|") {
+      dividerIndices.push(i);
+    }
+  }
+
+  const title = line.substring(dividerIndices[0] + 1, dividerIndices[1]);
+  const summary = line.substring(dividerIndices[1] + 2, dividerIndices[2]);
+  const markdownUmipLink = line.substring(dividerIndices[2] + 2, line.length);
```

Comment:
tiny nit (you don't need to change this if you prefer the way it's done now): it seems like this block might be a bit simpler by doing something like:

```ts
const [titleSection, summarySection, markdownUmipLinkSection] = line.split("|");
const title = titleSection.slice(1);
const summary = summarySection.slice(2);
const umipLink = (parseMarkdownUmipLink(markdownUmipLinkSection.slice(2));
```